tests/e2e_test/utils_test.go

```diff
@@ -102,6 +113,39 @@ func CheckKymaIsInState(ctx context.Context,
 	return nil
 }
 
+func CheckManifestIsInState(ctx context.Context,
+	manifestObjKey types.NamespacedName,
+	k8sClient client.Client,
+	expectedState declarative.State,
+) error {
+	manifest := &v1beta2.Manifest{}
+	if err := k8sClient.Get(ctx, manifestObjKey, manifest); err != nil {
+		return err
+	}
+	GinkgoWriter.Printf("manifest %v\n", manifest)
+	if manifest.Status.State != expectedState {
+		return fmt.Errorf("%w: expect %s, but in %s. Manifest CR: %#v",
+			errManifestNotInExpectedState, expectedState, manifest.Status.State, manifest)
+	}
+	return nil
+}
+
+func CheckManifestDeletionTimestamp(ctx context.Context,
+	manifestObjKey types.NamespacedName,
+	k8sClient client.Client,
+	expectedTimestampZero bool,
+) error {
+	manifest := &v1beta2.Manifest{}
+	if err := k8sClient.Get(ctx, manifestObjKey, manifest); err != nil {
+		return err
+	}
+	GinkgoWriter.Printf("manifest %v\n", manifest)
```

Comment:
this `GinkgoWriter` is just adding some debug information, please don't introduce them further. If the test fails, it's anyway will print out enough information for debug.